pubtools/_impl/pluggy.py

```diff
@@ -94,4 +94,22 @@ def task_context():
         pm.hook.task_stop(failed=failed)
 
 
+@hookspec(firstresult=True)
+def get_cert_key_paths_plugin(server_url):
+    """Get location of SSL certificates for a given service.
+
+    If there are multiple hook implementations and multiple values are returned, the first
```

Comment:
Minor - some more vertical whitespace would be nice.  This is a bunch of different information all in one paragraph (would all be rendered on a single line in a web browser reading the docs, if there's room for it).  Breaking it into multiple paragraphs would improve the readability.

Reply:
Added some whitespaces.